services/api/service.go

```diff
@@ -1348,6 +1557,23 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 		return
 	}
 
+	builderPubkey := payload.BuilderPubkey()
+	builderEntry, ok := api.blockBuildersCache[builderPubkey.String()]
+	if !ok {
+		log.Warnf("unable to read builder: %x from the builder cache, using low-prio and no collateral", builderPubkey.String())
```

Comment:
wouldn't `%s` be the right formatting argument here, instead of `%x`?

Reply:
good catch, yes!